packages/orbit-components/src/Itinerary/consts.ts

```diff
@@ -0,0 +1,85 @@
+export enum STATUS_TYPES {
+  CRITICAL = "critical",
+  INFO = "info",
+  SUCCESS = "success",
+  WARNING = "warning",
+  NEUTRAL = "neutral",
+}
+
+export enum PALETTE_COLORS {
```

Comment:
maybe I'm wrong and there is some plan I am not aware of yet, but I guess there won't be many changes in color palette naming (?). if yes, wouldn't be there same process as with the tokens and it can be potentially updated with codemod file? 🤔